api/src/certification/session/domain/usecases/integrate-cpf-processing-receipts.js

```diff
@@ -1,11 +1,13 @@
 import { logger } from '../../../../shared/infrastructure/utils/logger.js';
 
-const integrateCpfProccessingReceipts = async function ({ cpfReceiptsStorage }) {
+const integrateCpfProccessingReceipts = async function ({ cpfReceiptsStorage, cpfCertificationResultRepository }) {
   const cpfReceipts = await cpfReceiptsStorage.findAll();
   for (const cpfReceipt of cpfReceipts) {
     logger.debug('Treatment of CPF receipt %o', cpfReceipt);
     const cpfInfos = await cpfReceiptsStorage.getCpfInfosByReceipt({ cpfReceipt });
-    return cpfInfos;
+    for (const certificationCourseCpfInfos of cpfInfos) {
+      await cpfCertificationResultRepository.updateCpfInfos({ cpfInfos: certificationCourseCpfInfos });
+    }
```

Comment:
On pourrait parraleliser un minimum
```suggestion
    bluebird.map(cpfInfos, (certificationCourseCpfInfos) => 
     cpfCertificationResultRepository.updateCpfInfos({ cpfInfos: certificationCourseCpfInfos }),
     { concurrency: CONCURRENCY_HEAVY_OPERATIONS }
     )
    
```

Reply:
Done !